crates/wallet/src/lib.rs

```diff
@@ -210,7 +213,15 @@ where
         trace!(target: "rpc::wallet", ?request, "Serving wallet_sendTransaction");
 
         // validate fields common to eip-7702 and eip-1559
-        validate_tx_request(&request)?;
+        match validate_tx_request(&request) {
+            Ok(_) => {
+                self.inner.metrics.valid_send_transaction_calls.increment(1);
+            }
+            Err(err) => {
+                self.inner.metrics.invalid_send_transaction_calls.increment(1);
+                return Err(err.into());
+            }
```

Comment:
the tx is only valid if it passes all the checks before `EthTransactions::send_raw_transaction`